### DISCUSSION TITLE
Add to Original URL to search function

### DISCUSSION THREAD

**Discussion by bbrodka2** [mealie-recipes/mealie#3445] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Include field "Original URL" to search function on search page so we can quickly find all the recipes from a particular source

### Please provide a concise description of the feature that would resolve your issue.

There is currently no way to find recipies from a particular web sit via the user interface
example show all recipes from allrecipes.com

### Please consider and list out some caveats or tradeoffs made in your design decision

none that I can think of other then slight deley in search results for having to search an additional database field

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Kuchenpirat** [comment]
Hi, 
just trying to understand your motivation about that feature request here. 
Mealie was designed to eliminate the hassle of juggling various online recipe sources by consolidating them into one platform. Hence I don’t understand why you would want do split your recipes up by source for your search?
Could you perhaps provide a specific scenario where this functionality would prove beneficial?

**Reply by bbrodka2** [reply]
Mostly I would use it filter for data cleanp and edit after a bulk import.
This could also be achived by having an option to set a defalt tag durring bulk import, which I could then deleter after I reviewed the recipe

curently there is no way to filter recipes by sours or newly imported (Other then sort by date added), but I may not get to them all in one sitting, do a way to filter by a automaticly created tag or url serch would be helpful

this is not something I would use while actually looking for a recipe, more for administrative tasks

Thanks for your hard work, I've just installed on unraid this last weekend, and am really impressed with what youve done

**Reply by Kuchenpirat** [reply]
Ah ok, that makes a lot of sense. I think building a whole search around that seems a bit overkill. We have a task written up, that would offer a more flexible solution which would allow you to add add automated labeling which would propably be enough (#1106)

But i don't think much work in this direction has been completed.

**Reply by bbrodka2** [reply]
Yes, that would work, thanks
BTW posibly the ability to add multiple tags seperated by comanas
tag1,tag2,etc
would be good

**Comment by burgerga** [comment]
I really like this feature in Whisk / Samsung Food, where the source is listed for every recipe (and my face and name for my own recipes):
![image](https://github.com/mealie-recipes/mealie/assets/907362/ba3835db-fe1d-4014-b0f3-92c01b627540)

Sometimes I know better what recipe source (website or cookbook) I want to use, than what I want to eat (e.g., my weekday recipes are very often from ah.nl, but in the weekend I might want to cook something a little more special, so for example I want something from jamieoliver.com).

My current solution is using tags, and then creating a cookbook based on that tag, but it would be nice if this would be automatic :)
